<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="edit_profile_label">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit Profile</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstname">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>151</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter FirstName</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastname">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>151</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter LastName</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="city">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>221</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Your City</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>348</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phonenumber">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>291</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter PhoneNumber</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="birthday">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>291</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Birthday</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>221</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fanme_checkbox">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>154</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Firstname</string>
    </property>
   </widget>
   <widget class="QLabel" name="info_label">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>90</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select any of the information you want to change!</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lastname_checkbox">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>154</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LastName</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="city_checkbox">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>224</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>City</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="password_checkbox">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>224</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="birthday_checkbox">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>294</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Birthday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="phonenumber_checkbox">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>294</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PhoneNumber</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="email_checkbox">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>357</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_btn_editform">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>416</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Return to Setting</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
